source/SkiaSharp.Views.Forms/SkiaSharp.Views.Forms.UWP/SKTouchHandler.cs

```diff
@@ -127,22 +127,27 @@ private bool CommonHandler(object sender, SKTouchAction touchActionType, Pointer
 			var skPoint = scalePixels(windowsPoint.X, windowsPoint.Y);
 
 			var mouse = GetMouseButton(pointerPoint);
-			var device = GetTouchDevice(evt);
+			var device = GetTouchDevice(evt, pointerPoint);
 
 			var wheelDelta = pointerPoint?.Properties?.MouseWheelDelta ?? 0;
+			var pressure = pointerPoint?.Properties?.Pressure ?? 0.5;
```

Comment:
This should be `?? 1` because 1 is the default pressure for `SKTouchEventArgs`. Also this change would fix the compiler error.
[pr1239fix.patch](https://github.com/mono/SkiaSharp/files/15142696/pr1239fix.patch)